dotnet/src/Skills/Skills.Planning/Skills.Planning.csproj

```diff
@@ -0,0 +1,40 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <!-- THIS PROPERTY GROUP MUST COME FIRST -->
+    <AssemblyName>Microsoft.SemanticKernel.Skills.Planning</AssemblyName>
```

Comment:
```Suggestion
    <AssemblyName>Microsoft.SemanticKernel.Planning</AssemblyName>
```

Reply:
the planner is a skill, I would follow the existing naming convention

Reply:
To me it's more of a SkillGenerator or FunctionGenerator. It could be wrapped in a skill but I don't think we should be doing that right now. I'd like to understand the scenario more if we think urgent for build.

Reply:
This goes back to the design and message we've been giving since the start: planners and memory are skills. For instance we will have planner skills discovering other planners. These building blocks can be combined in many ways to achieve higher level concepts (like function generator) but they are still skills and can be composed on many ways. If we decide that they are something else, then the design will diverge and they will become unusable in scenarios where one wants to compose skills including these planners.